x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/components/connection_status/connection_status.js

```diff
@@ -54,6 +59,13 @@ export function ConnectionStatus({ isConnected }) {
           {message}
         </EuiText>
       </EuiFlexItem>
+      <EuiFlexItem>
+        <EuiIconTip
+          type="iInCircle"
+          color="subdued"
+          content={seedNodeTooltip}
+        />
+      </EuiFlexItem>
```

Comment:
We can nest the X icon and "Not connected" text inside another flex group, to get the tooltip to center over the icon like this:

![image](https://user-images.githubusercontent.com/1238659/51578272-cfec0500-1e71-11e9-92d3-d0b81db10688.png)


```jsx
<EuiFlexGroup gutterSize="s" alignItems="center">
  <EuiFlexItem grow={false}>
    <EuiFlexGroup gutterSize="s" alignItems="center">
      <EuiFlexItem grow={false}>
        {icon}
      </EuiFlexItem>

      <EuiFlexItem>
        <EuiText>
          {message}
        </EuiText>
      </EuiFlexItem>
    </EuiFlexGroup>
  </EuiFlexItem>

  <EuiFlexItem grow={false}>
    <EuiIconTip
      color="subdued"
      content={seedNodeTooltip}
    />
  </EuiFlexItem>
</EuiFlexGroup>
```

Reply:
Updated, here's a screenshot:

<img width="685" alt="screen shot 2019-01-23 at 10 56 02 am" src="https://user-images.githubusercontent.com/163306/51619168-c0b29900-1efd-11e9-92be-390718cd9ec7.png">

Reply:
Here's an updated screenshot from local test
<img width="654" alt="screen shot 2019-01-23 at 11 03 14 am" src="https://user-images.githubusercontent.com/163306/51619577-aaf1a380-1efe-11e9-82c4-0ef4b903177f.png">